launch/launch/__init__.py

```diff
@@ -38,7 +40,9 @@
     'actions',
     'conditions',
     'events',
+    'launch_frontend',
```

Comment:
@ivanpauno can't we just call it `frontend`? To go from `launch.launch_frontend` to `launch.frontend`.